src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/LoggerExtensions.cs

```diff
@@ -245,6 +251,11 @@ public static void LogFunctionRetryAttempt(this ILogger logger, TimeSpan nextDel
             _logFunctionRetryAttempt(logger, nextDelay, attemptCount, maxRetryCount, null);
         }
 
+        public static void LogFunctionRetryFailed(this ILogger logger, int attemptCount, IDelayedException result)
```

Comment:
Let's rename to LogFunctionRetriesFailed to make this clearer (we're not talking about a single retry attempt)